### DISCUSSION TITLE
Some random benchmark puts uWebSocket as 3rd fastest

### DISCUSSION THREAD

**Discussion by ddevienne** [uNetworking/uWebSockets#1651] [General]
I know you scoff of these, but just in case you haven't seen it, feel free to tear that one down or ignore it. Just FYI.

https://c410-f3r.github.io/thoughts/the-fastest-websocket-implementation/

**Comment by uNetworkingAB** [comment]
If you do more than two seconds of review you can see that the guy who put this wtx project as **significantly** faster than all others, also made this wtx project.... 4 days ago (initial commit).

Just because a document looks well presented doesn't mean its content is of any more value than that of a pissed down grafitti:d wall in downtown Philadelphia.

Most people have no idea how to benchmark things properly, esp. not fast things.
